packages/venia-ui/lib/components/CartPage/ProductListing/quantity.js

```diff
@@ -30,40 +31,43 @@ export const QuantityFields = props => {
     } = talonProps;
 
     return (
-        <div className={classes.root}>
-            <label className={classes.label} htmlFor={itemId}>
-                {label}
-            </label>
-            <button
-                aria-label={'Decrease Quantity'}
-                className={classes.button_decrement}
-                disabled={isDecrementDisabled}
-                onClick={handleDecrement}
-                type="button"
-            >
-                <Icon classes={iconClasses} src={MinusIcon} size={22} />
-            </button>
-            <TextInput
-                aria-label="Item Quantity"
-                classes={{ input: classes.input }}
-                field="quantity"
-                id={itemId}
-                inputMode="numeric"
-                mask={maskInput}
-                min={min}
-                onBlur={handleBlur}
-                pattern="[0-9]*"
-            />
-            <button
-                aria-label={'Increase Quantity'}
-                className={classes.button_increment}
-                disabled={isIncrementDisabled}
-                onClick={handleIncrement}
-                type="button"
-            >
-                <Icon classes={iconClasses} src={PlusIcon} size={20} />
-            </button>
-        </div>
+        <Fragment>
+            <div className={classes.root}>
+                <label className={classes.label} htmlFor={itemId}>
+                    {label}
+                </label>
+                <button
+                    aria-label={'Decrease Quantity'}
+                    className={classes.button_decrement}
+                    disabled={isDecrementDisabled}
+                    onClick={handleDecrement}
+                    type="button"
+                >
+                    <Icon classes={iconClasses} src={MinusIcon} size={22} />
+                </button>
+                <TextInput
+                    aria-label="Item Quantity"
+                    classes={{ input: classes.input }}
+                    field="quantity"
+                    id={itemId}
+                    inputMode="numeric"
+                    mask={maskInput}
+                    min={min}
+                    onBlur={handleBlur}
+                    pattern="[0-9]*"
+                />
+                <button
+                    aria-label={'Increase Quantity'}
+                    className={classes.button_increment}
+                    disabled={isIncrementDisabled}
+                    onClick={handleIncrement}
+                    type="button"
+                >
+                    <Icon classes={iconClasses} src={PlusIcon} size={20} />
+                </button>
+            </div>
+            {message && <Message>{message}</Message>}
```

Comment:
We have a preference to avoid inline expressions like this; consider extracting this to a variable.